public/app/features/scenes/scenes/index.tsx

```diff
@@ -10,32 +10,41 @@ import { getNestedScene } from './nested';
 import { getSceneWithRows } from './sceneWithRows';
 import { getVariablesDemo } from './variablesDemo';
 
-export function getScenes(): Scene[] {
+interface SceneDef {
+  title: string;
+  getScene: () => Scene;
+}
+export function getScenes(): SceneDef[] {
   return [
-    getFlexLayoutTest(),
-    getScenePanelRepeaterTest(),
-    getNestedScene(),
-    getSceneWithRows(),
-    getGridLayoutTest(),
-    getGridWithRowLayoutTest(),
-    getGridWithMultipleData(),
-    getGridWithMultipleTimeRanges(),
-    getMultipleGridLayoutTest(),
-    getVariablesDemo(),
+    { title: 'Flex layout test', getScene: getFlexLayoutTest },
+    { title: 'Panel repeater test', getScene: getScenePanelRepeaterTest },
+    { title: 'Nested Scene demo', getScene: getNestedScene },
+    { title: 'Scene with rows', getScene: getSceneWithRows },
+    { title: 'Grid layout test', getScene: getGridLayoutTest },
+    { title: 'Grid with row layout test', getScene: getGridWithRowLayoutTest },
+    { title: 'Grid with rows and different queries', getScene: getGridWithMultipleData },
+    { title: 'Grid with rows and different queries and time ranges', getScene: getGridWithMultipleTimeRanges },
+    { title: 'Multiple grid layouts test', getScene: getMultipleGridLayoutTest },
+    { title: 'Variables', getScene: getVariablesDemo },
   ];
 }
 
 const cache: Record<string, Scene> = {};
 
-export function getSceneByTitle(title: string) {
+export function getSceneByTitle(title: string, standalone = true) {
   if (cache[title]) {
+    if (cache[title].state.standalone !== standalone) {
+      cache[title].setState({ standalone });
```

Comment:
This happens in ScenePage render, think for dev purposes it's fine, but in general,  setting Scene's state during render is an anti-pattern.

Reply:
yea, this is not ok.  will likely leed to react errors

Reply:
I guess it's ok as this scene object has not activated (mounted yet)

Reply:
Changed that - now there's a simple EmbeddedScene alternative